build.gradle

```diff
@@ -24,7 +23,6 @@ buildscript {
         classpath 'net.ltgt.gradle:gradle-apt-plugin:0.5' // for non-android projects
```

Comment:
@digitalbuddha There is a comment about "non-android projects" so I didn't touch. But let me try clean it and push to see if it still builds.